src/modules/uxrce_dds_client/uxrce_dds_client.h

```diff
@@ -159,6 +164,15 @@ class UxrceddsClient : public ModuleBase<UxrceddsClient>, public ModuleParams
 	SendTopicsSubs *_subs{nullptr};
 	RcvTopicsPubs *_pubs{nullptr};
 
+	uxrObjectId _participant_id;
+
+	uint8_t _output_reliable_stream_buffer[BUFFER_SIZE] {};
+	uint8_t _output_data_stream_buffer[2048] {};
+	uint8_t _input_reliable_stream_buffer[BUFFER_SIZE] {};
```

Comment:
Could you check if you can reduce the stack size of the thread now. I think these ~4kB are now moved to the head instead of the stack.

Reply:
Good idea, reduced the stack usage.